log_storage/models.py

```diff
@@ -15,23 +15,21 @@ class BaseLog(models.Model):
     class Meta:
         abstract = True
 
-    def get_filename(self):
+    def _get_desired_filename(self):
         import os, time, random
         from django.conf import settings
         path = getattr(settings, 'LOG_DATA_DIR', 'logs')
-        if not self.filename:
-            self.filename = '_'.join(filter(bool, [self.prefix,
+        filename = '_'.join(filter(bool, [self.prefix,
                     time.strftime('%Y%m%d_%H%M%S'),
                     ''.join([random.choice('0123456789') for i in range(0, 5)]),
                     self.suffix]))
```

Comment:
We could simplify this a bit more since we're not using a Django storage, if the file already exists it will add the random characters to be able to persist it without conflicts.
Maybe just keep (prefix + time + suffix)?